src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
Do not create a new `Random` instance each time, make it a class-level variable.
Also, check the documentation of the nextInt() method, bound parameter there is exclusive, which means that right now the maximum possible number that we can get is 99